src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -708,6 +709,16 @@ public class SimpleEvents {
 					.since("2.7")
 					.requiredPlugins("Paper 1.16+");
 		}
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.EndermanEscapeEvent")) {
+			Skript.registerEvent("Enderman Escape", SimpleEvent.class, EndermanEscapeEvent.class, "enderman escap(e[d]|ing)")
+				.description("Called when an enderman escapes by teleporting away from its location.")
+				.examples(
+					"on enderman escape:",
+						"\tbroadcast \"%event-entity% has escaped due to %escape reason%!\""
+				)
+				.since("INSERT VERSION")
+				.requiredPlugins("Paper");
```

Comment:
```suggestion
					.description("Called when an enderman escapes by teleporting away from its location.")
					.examples(
						"on enderman escape:",
							"\tbroadcast \"%event-entity% has escaped due to %escape reason%!\""
					)
					.requiredPlugins("Paper")
					.since("INSERT VERSION");
```